cfgov/v1/atomic_elements/organisms.py

```diff
@@ -506,7 +506,7 @@ class VideoPlayer(blocks.StructBlock):
         label='YouTube Embed URL',
         default='https://www.youtube.com/embed/',
         required=True,
-        regex=r'^https:\/\/www\.youtube\.com\/embed\/[a-zA-Z0-9]+([/]?)$',
+        regex=r'^https:\/\/www\.youtube\.com\/embed\/[a-zA-Z0-9\-]+([/]?)$',
```

Comment:
Thanks, Chris. Steven also pointed that out in an off-thread chat. We decided to just make it significantly more forgiving for the time being, with the intention of refactoring the rendering macro in the near future to accept any valid YouTube URL, or even just the ID.